lms/djangoapps/learner_home/serializers.py

```diff
@@ -561,6 +561,7 @@ class EnterpriseDashboardSerializer(serializers.Serializer):
     label = serializers.CharField(source="name")
     url = serializers.SerializerMethodField()
     uuid = serializers.UUIDField()
+    isLearnerPortalEnabled = serializers.BooleanField(source="enable_learner_portal")
```

Comment:
Convention in these serializers is to return camelCased  properties instead of snake_cased, so I'm following along with that convention here.